ffi/diplomat/tests/missing_apis.txt

```diff
@@ -22,6 +22,20 @@ icu::collections::codepointinvliststringlist::CodePointInversionListAndStringLis
 icu::collections::codepointinvliststringlist::CodePointInversionListAndStringList::try_from#FnInStruct
 icu::collections::codepointinvliststringlist::CodePointInversionListAndStringListError#Enum
 icu::collections::codepointinvliststringlist::Error#Enum
+icu::compactdecimal::CompactDecimalError#Enum
+icu::compactdecimal::CompactDecimalFormatter#Struct
+icu::compactdecimal::CompactDecimalFormatter::compact_exponent_for_magnitude#FnInStruct
+icu::compactdecimal::CompactDecimalFormatter::format_compact_decimal#FnInStruct
+icu::compactdecimal::CompactDecimalFormatter::format_i64#FnInStruct
+icu::compactdecimal::CompactDecimalFormatter::try_new_long_unstable#FnInStruct
+icu::compactdecimal::CompactDecimalFormatter::try_new_short_unstable#FnInStruct
+icu::compactdecimal::provider::CompactDecimalPatternDataV1#Struct
+icu::compactdecimal::provider::Count#Enum
+icu::compactdecimal::provider::CountULE#Struct
+icu::compactdecimal::provider::LongCompactDecimalFormatDataV1Marker#Struct
+icu::compactdecimal::provider::Pattern#Struct
```

Comment:
also please undo the changes to this file

Reply:
When I undo, the test fails.

Reply:
Oh that's because you changed the other deptest (which was the right thing to do)

I'm talking about the ffi coverage test: it should allowlist `icu::compactdecimal`